src/Compilers/CSharp/Test/Semantic/Semantics/InterceptorsTests.cs

```diff
@@ -322,9 +426,44 @@ public static class Interceptor
 
         var comp = CreateCompilation(new[] { source0, source1, source2, s_attributesSource });
         comp.VerifyEmitDiagnostics(
-            // Interceptor.cs(15,6): error CS27015: Cannot intercept a call in file with path 'Program.cs' because multiple files in the compilation have this path.
+            // Interceptor.cs(15,25): error CS27015: Cannot intercept a call in file with path 'Program.cs' because multiple files in the compilation have this path.
             //     [InterceptsLocation("Program.cs", 5, 11)]
-            Diagnostic(ErrorCode.ERR_InterceptorNonUniquePath, @"InterceptsLocation(""Program.cs"", 5, 11)").WithArguments("Program.cs").WithLocation(15, 6));
+            Diagnostic(ErrorCode.ERR_InterceptorNonUniquePath, @"""Program.cs""").WithArguments("Program.cs").WithLocation(15, 25));
+    }
+
+    [Fact]
+    public void DuplicateLocation_01()
+    {
+        var source = """
+            using System.Runtime.CompilerServices;
+
+            C.M();
+
+            class C
+            {
+                [Interceptable]
+                public static void M() { }
+            }
+
+            class D
+            {
+                [InterceptsLocation("Program.cs", 3, 3)]
+                public static void M1() { }
+
+                [InterceptsLocation("Program.cs", 3, 3)]
+                public static void M2() { }
+            }
+            """;
+
+        var comp = CreateCompilation(new[] { (source, "Program.cs"), s_attributesSource });
+        comp.VerifyDiagnostics();
+        comp.VerifyEmitDiagnostics(
+            // Program.cs(13,6): error CS27016: The indicated call is intercepted multiple times.
```

Comment:
nit: In terms of usability, this diagnostic isn't the best. My first question will be where are the other interceptions. If the error list is small enough, they I can find it that way.
This will be improved/resolved once we have some IDE experience to list the interceptions for a given invocation (maybe in QuickInfo or FAR).

Reply:
We could include the display of the intercepted method, perhaps.